rotkehlchen/tests/unit/test_gnosis_inquirer.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+
+from rotkehlchen.tests.utils.factories import make_evm_address
+from rotkehlchen.tests.utils.gnosis import GNOSIS_NODES_PARAMETERS_WITH_PRUNED_AND_NOT_ARCHIVED
+
+
+@pytest.mark.parametrize(*GNOSIS_NODES_PARAMETERS_WITH_PRUNED_AND_NOT_ARCHIVED)
+@pytest.mark.parametrize('gnosis_accounts', [[make_evm_address()]])  # to connect to nodes
+def test_gnosis_nodes_prune_and_archive_status(
+        gnosis_manager_connect_at_start,
+        gnosis_inquirer,
+):
+    """Checks that connecting to a set of gnosis nodes, the capabilities of those nodes are known and stored.
+    This test is sort of fast and is not VCRed due to the randomness of the connection
+    to the nodes not being able to be replicated in a reproducible way in the cassetes.
+    """  # noqa: E501
```

Comment:
Rearrange a bit to get rid of that noqa. Note: not sure my rearrangement does it but you get the point

```suggestion
    """Checks that connecting to a set of gnosis nodes, the capabilities of those nodes
    are known and stored. This test is sort of fast and is not VCRed due to the randomness of
    the connection to the nodes not being able to be replicated in a reproducible way in the cassetes.
    """ 
```